framework/src/actions/AddAuxVariableAction.C

```diff
@@ -28,7 +28,10 @@ AddAuxVariableAction::AddAuxVariableAction(const InputParameters & params)
 MooseEnum
 AddAuxVariableAction::getAuxVariableFamilies()
 {
-  return MooseEnum("LAGRANGE MONOMIAL SCALAR LAGRANGE_VEC MONOMIAL_VEC", "LAGRANGE", true);
+  return MooseEnum("LAGRANGE MONOMIAL SCALAR LAGRANGE_VEC MONOMIAL_VEC L2_HIERARCHIC "
+                   "L2_HIERARCHIC_VEC L2_LAGRANGE L2_LAGRANGE_VEC",
```

Comment:
so does auxkernels work with these variables? I remember some trouble while working on projectionAux

Reply:
they should be able to. That doesn't mean I'm going to guarantee that all aux kernels work with them